src/python/py/models/builder.py

```diff
@@ -665,6 +676,18 @@ def make_tanh(self, name, root_input, dtype, shape):
         self.make_value_info(output, dtype, shape=shape)
 
     def make_matmul(self, matmul, basename, root_input, **kwargs):
+        print("Matmul attr = ", matmul)
+        if hasattr(matmul, "base_layer"):
+            # For LoRA `MatMul`
+            print("Making lora matmul")
+            return self.make_matmul_lora(matmul, basename, root_input, **kwargs)
+        # elif self.matmul_attrs["use_qdq"]:
+        #     return self.make_matmul_qdq(matmul, basename, root_input, **kwargs)
```

Comment:
Can we remove the commented out code for QDQ since it's supported here?

https://github.com/microsoft/onnxruntime-genai/blob/0f59a900568cf65d74554cc34027648989f4ab1c/src/python/py/models/builder.py#L667-L676

Reply:
rebase did not happen properly. Fixing it